manuscript/060_Statement_First_Programming.md

```diff
@@ -135,39 +134,41 @@ var configuration = new XmlConfiguration(customFixture);
 ...
 ```
 
-And it passes -- cool... not. Ok, what is wrong with this? Nothing big, unless you read the source code of XmlConfiguration class carefully. Inside, you can see, how the xml string is stored:
+And it passes -- cool... not. Ok, what is wrong with this? Nothing big, unless you read the source code of XmlConfiguration class carefully. Inside, you can see, how the XML string is stored:
 
 ```csharp
 private static string xmlText; //note the static keyword!
 ```
 
-What the...? Well, well, here is what happened: the author of this class coded in a�small little optimization. He thought: �In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I�can save some CPU cycles and I/O operations by reading it only once when the first object is created. Another created object will just use the same XML!". Good for him, not so good for you. Why? Because (unless your Statement is evaluated prior to being fulfilled), your custom xml string will never actually be used!
+What the...? Well, well, here is what happened: the author of this class applied a small optimization. He thought: “In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I can save some CPU cycles and I/O operations by reading it only once when the first object is created. Later objects will just use the same XML!". Good for him, not so good for you. Why? Because your custom XML string will never actually be used! (Unless your Statement is evaluated prior to being fulfilled.)
```

Comment:
What would you say about:
"never actually be used! (Unless your Statement is evaluated prior to being fulfilled.)"
to 
"never actually be used (Unless your Statement is evaluated prior to being fulfilled)!"